app/main.py

```diff
@@ -2,5 +2,16 @@
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    results = {}
+
+    def wrapper(*args) -> str:
+        if args in results:
+            print("Getting from cache")
+            return results[args]
+        else:
```

Comment:
There is no need for `else` since the code will not execute after the `return` statement anyway. Remove redundant `else`.